### PR TITLE
fix(posts): change stylesheet name in post. (2019-08-09-getting-started.md)

### PR DESCRIPTION
**Pull request by dennykorsukewitz**
## Type of change

- [x] Documentation update

## Description

After this change, the corresponding post (2019-08-09-getting-started.md) must also be adjusted.

https://github.com/cotes2020/jekyll-theme-chirpy/commit/806fa3aa1a7444186e1db4fe693886450e351997

## Additional context

Nothing else...

### CODE REVIEW

File: _posts/2019-08-09-getting-started.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Social contact options are displayed at the bottom of the sidebar. You can turn
 
 ### Customizing Stylesheet
 
-If you need to customize the stylesheet, copy the theme's `assets/css/style.scss`{: .filepath} to the same path on your Jekyll site, and then add the custom style at the end of it.
+If you need to customize the stylesheet, copy the theme's `assets/css/jekyll-theme-chirpy.scss`{: .filepath} to the same path on your Jekyll site, and then add the custom style at the end of it.
 
 Starting with version `4.1.0`, if you want to overwrite the SASS variables defined in `_sass/addon/variables.scss`{: .filepath}, copy the main sass file `_sass/jekyll-theme-chirpy.scss`{: .filepath} into the `_sass`{: .filepath} directory in your site's source, then create a new file `_sass/variables-hook.scss`{: .filepath} and assign new value.
```
Comment (cotes2020) [inline _posts/2019-08-09-getting-started.md:75]:
```suggestion
Starting with version `6.2.0`, if you want to overwrite the SASS variables defined in `_sass/addon/variables.scss`{: .filepath}, copy the main sass file `_sass/main.scss`{: .filepath} into the `_sass`{: .filepath} directory in your site's source, then create a new file `_sass/variables-hook.scss`{: .filepath} and assign new value.
```

Comment (cotes2020) [reply _posts/2019-08-09-getting-started.md:75]: The `variables-hook` section also needs to be updated

**Review by cotes2020 — CHANGES_REQUESTED** [review]
Thanks for the additions to the docs, still a bit short of this PR becoming complete :)

File: _posts/2019-08-09-getting-started.md (hunk 1 above)
Comment (dennykorsukewitz) [reply _posts/2019-08-09-getting-started.md:75]: True. I'll look at it later. 👍🏼

**Comment by dennykorsukewitz** [comment]
Ahh now you were faster than me.
😁👍👍

**Pull request merged**